jenkins_pipelines/environments/manager-4.3-dev-acceptance-tests-NUE

```diff
@@ -14,6 +14,7 @@ node('sumaform-cucumber') {
             choice(name: 'sumaform_backend', choices: ['libvirt', 'aws'], description: 'Sumaform backend to be used (see https://github.com/uyuni-project/sumaform#backend-choice)'),
             choice(name: 'terraform_bin', choices: ['/usr/bin/terraform'], description: 'Terraform binary path'),
             choice(name: 'terraform_bin_plugins', choices: ['/usr/bin'], description: 'Terraform plugins path'),
+            choice(name: 'terraform_parallelism', choices: ['20'], description: 'Define the number of parallel resource operations for terraform'),
```

Comment:
Instead of using an unique choice, that will not let as parametrize this value when we run the pipeline... better use 20 as default value of a string.

```suggestion
             string(name: 'terraform_parallelism', defaultValue: '20', description: 'Define the number of parallel resource operations for terraform'),
```